libs/zzz/formula/src/data/common/daze.ts

```diff
@@ -0,0 +1,31 @@
+import { prod, sum } from '@genshin-optimizer/pando/engine'
+import type { TagMapNodeEntries } from '../util'
+import { enemy, own, ownBuff, percent } from '../util'
+
+const data: TagMapNodeEntries = [
+  // Attacker Daze Multiplier
+  ownBuff.formula.daze_mult_.add(
+    sum(percent(1), own.final.dazeInc_, prod(-1, own.final.dazeRed_))
+  ),
+  // Enemy Daze Resistance Multiplier
+  ownBuff.formula.enemyDazeRes_mult_.add(
+    sum(percent(1), prod(-1, enemy.common.dazeRes_))
+  ),
+  // Enemy Daze Taken Multiplier
+  ownBuff.formula.enemyDazeTaken_mult_.add(
+    sum(percent(1), enemy.common.dazeInc_, prod(-1, enemy.common.dazeRed_))
```

Comment:
Should we add this to enemy editor? or future PR?

Reply:
the expectation was that we would merge this before frzyc would work on the enemy editor, now i guess we should